src/test/java/au/org/democracydevelopers/raireservice/service/GenerateAssertionsOnNSWTests.java

```diff
@@ -75,7 +80,7 @@ public class GenerateAssertionsOnNSWTests {
    * Difficulties are taken from raire-java::src/test/java/au/org/democracydevelopers/raire/TestNSW
    * which in turn tests against raire-rs.
    * Winners are taken from the New South Wales official election results at
-   * https://pastvtr.elections.nsw.gov.au/LG2101/status/mayoral
+   * pastvtr.elections.nsw.gov.au/LG2101/status/mayoral
```

Comment:
I think the https is better, just in case someone copy-pastes it, which they might.